sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -12368,6 +12451,24 @@
           "tree": false,
           "lazyLoad": false,
           "id": "e25p0uk"
+        },
+        {
+          "input": true,
+          "tableView": true,
+          "key": "disabilityStatusValue",
```

Comment:
This key could be named as `studentProfileDisabilityStatusValue` or maybe the label could be changed. The logic seems ok but it wasn't clear at the first time I saw it.

Reply:
I agree on the point the label could be updated to say, `Student profile disability status value`

Reply:
Thanks, this should now be updated on all the forms.